### PR TITLE
Gives Legion and NCR ID consoles

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

- Legion and NCR get unique ID consoles. Only command roles can use these and only few options are available, custom not included.
- Option to view crew manifest on console was also removed due to admin request.
- Wastelander NCR/Legion loadouts get citizen "IDs". These have no access but could be a good source of RP.

- Some Legion doors are now ID-locked.

## Why It's Good For The Game

Field promotions, citizenships and more. Sounds like a good roleplay mechanic to me.
There is really no mechanic benefits to this.

On the other hand, giving Legion ID-locked doors will certainly help against random shitters that run into the camp and snatch shit from armory.

## Changelog
:cl:
add: Gave NCR and Legion ID consoles and a bunch of spare IDs.
tweak: Legion armory doors are now ID-locked.
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/ncr.dm [hunk 1]
```diff
@@ -1103,3 +1103,22 @@ Weapons		Service Rifle, Rockwell, 9mm pistol, all good.
...
+/datum/outfit/job/ncr/f13ncrcitizen
+	name = "NCR Citizen (Role)"
+	uniform = /obj/item/clothing/under/f13/ncrcaravan
+	shoes = /obj/item/clothing/shoes/f13/tan
+	head = /obj/item/clothing/head/f13/cowboy
+	gloves = /obj/item/clothing/gloves/color/brown
+	id = /obj/item/card/id/dogtag/town/ncr
+	l_hand = /obj/item/gun/ballistic/rifle/mag/varmint
+	backpack_contents = list(
+		/obj/item/ammo_box/magazine/m556/rifle=2)
```
Comment (Rohesie) [inline code/modules/jobs/job_types/ncr.dm:1124]:
```suggestion
	backpack_contents = list(
		/obj/item/ammo_box/magazine/m556/rifle=2,
		)
```
* Make use of trailing commas.

File: code/game/machinery/computer/card.dm [hunk 2]
```diff
@@ -624,7 +457,41 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
...
+		"NCR Citizen",
+		"NCR Trooper",
+		"NCR Corporal",
+		"NCR Combat Medic",
+		"NCR Combat Engineer",
+		"NCR Sergeant",
+	)
+
+/obj/machinery/computer/card/ncr/ui_interact(mob/user)
+	var/list/job_req = list("NCR Colonel", "NCR Captain", "NCR Lieutenant")
```
Comment (Rohesie) [inline code/game/machinery/computer/card.dm:474]: Can you put this as an object variable, so it can be edited and redefined?

Comment (EgorDinamit) [reply code/game/machinery/computer/card.dm:474]: Fair. Will do just that.

File: code/game/machinery/computer/card.dm [hunk 3]
```diff
@@ -624,7 +463,29 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
...
+	// The job list is here so only specific titles are allowed.
+	job_list = list(
+		"NCR Citizen",
+		"NCR Trooper",
+		"NCR Corporal",
+		"NCR Combat Medic",
+		"NCR Combat Engineer",
+		"NCR Sergeant",
+	)
+	job_req = list("NCR Colonel", "NCR Captain", "NCR Lieutenant")
```
Comment (Rohesie) [inline code/game/machinery/computer/card.dm:478]:
```suggestion
	job_req = list(
		"NCR Colonel",
		"NCR Captain",
		"NCR Lieutenant",
	)
```

File: code/game/machinery/computer/card.dm [hunk 4]
```diff
@@ -624,7 +463,29 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
...
+	circuit = /obj/item/circuitboard/computer/card/legion
+	// tl;dr - We don't need more Centurions.
+	job_list = list(
+		"Legion Slave",
+		"Legion Citizen",
+		"Recruit Legionnaire",
+		"Prime Legionnaire",
+		"Veteran Legionnaire",
+	)
+	job_req = list("Legate", "Legion Centurion", "Legion Veteran Decanus")
```
Comment (Rohesie) [inline code/game/machinery/computer/card.dm:491]:
```suggestion
	job_req = list(
		"Legate",
		"Legion Centurion",
		"Legion Veteran Decanus",
		)
```

**Pull request merged**